routes/api/users.js

```diff
@@ -1,13 +1,22 @@
 const express = require("express");
+const multer = require("multer");
+const path = require("path");
+const { writeFile } = require("fs/promises");
 
 require("../../auth/strategy");
 const auth = require("../../auth/auth");
-const { logout, loginUser, registerUser } = require("../../models/users");
+const {
+	logout,
+	loginUser,
+	registerUser,
+	updateAvatar,
+} = require("../../models/users");
 const { userValidator } = require("../../validators/usersValidator");
 
 const router = express.Router();
+const upload = multer();
 
-router.post("/signup", async (req, res, next) => {
+router.post("/signup", upload.single("picture"), async (req, res, next) => {
```

Comment:
Przy signup nie robimy upload, to jest miejsce do rejestracji niż do customizacji user.